File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.py
while(True):
  name, age, weight = input().split()

  if (name == '#' and int(age) == 0 and int(weight) == 0):
    break
  
  if (int(age) < 18 and int(weight) < 80):
    print(f"{name} Junior")
  else:
    print(f"{name} Senior")

File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.c
#include <stdio.h>
#include <string.h>

int main(void) {  
   int age, weight;
   char name[11];

   while(1){
      scanf("%s %d %d", name, &age, &weight);
      if (strcmp(name, "#") == 0 && age == 0 && weight == 0){
         break;
      }

      if (age < 18 && weight < 80){
         printf("%s Junior\n", name);
      }
      else{
         printf("%s Senior\n", name);
      }
   }

   
   return 0;
}